From 854e1f23218b8e81db2c05728f8e21cd09dd69eb Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Tue, 17 Dec 2013 14:01:34 -0500
Subject: [PATCH 71/78] Ticket 47620 - Fix missing left bracket

https://fedorahosted.org/389/ticket/47620
(cherry picked from commit 3a4d39e166449177c85b92af8b47c5c6848c4d02)
(cherry picked from commit b143477e186734333b6cfae615484b886b204e0e)
---
 ldap/servers/plugins/replication/repl5_agmtlist.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/replication/repl5_agmtlist.c b/ldap/servers/plugins/replication/repl5_agmtlist.c
index 6e8b82c..9e0291f 100644
--- a/ldap/servers/plugins/replication/repl5_agmtlist.c
+++ b/ldap/servers/plugins/replication/repl5_agmtlist.c
@@ -245,7 +245,7 @@ agmtlist_modify_callback(Slapi_PBlock *pb, Slapi_Entry *entryBefore, Slapi_Entry
 	for (i = 0; NULL != mods && NULL != mods[i]; i++)
 	{
 		slapi_ch_free_string(&val);
-		if (mods[i]->mod_bvalues && mods[i]->mod_bvalues[0])
+		if (mods[i]->mod_bvalues && mods[i]->mod_bvalues[0]){
 			val = slapi_berval_get_string_copy (mods[i]->mod_bvalues[0]);
 		}
 		if (slapi_attr_types_equivalent(mods[i]->mod_type, type_nsds5ReplicaInitialize))
-- 
1.8.1.4

